BUILD.bazel

```diff
@@ -562,6 +563,25 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "numpy",
+    srcs = [
+        "stablehlo/reference/NumPy.cpp",
+    ],
+    hdrs = [
+        "stablehlo/reference/NumPy.h",
+    ],
+    includes = ["."],
+    deps = [
+        ":reference_index",
+        ":reference_tensor",
+        "@llvm-project/llvm:Support",
+        "@llvm-project/mlir:IR",
+        "@llvm-project/mlir:QuantOps",
+        "@llvm-project/mlir:Support",
```

Comment:
Missing a slash. Bazel GH actions currently failing on this

```
"@llvm-project//llvm:Support"
```

Reply:
Look like this was missed? Bazel build still failing since single slash is used.